rh-push-to-registry-redhat-io/release-resources/release-plan-admission.yaml

```diff
@@ -6,6 +6,27 @@ metadata:
   name: rh-push-to-registry-redhat-io-test-rpa
   namespace: managed-release-team-tenant
 spec:
-  application: rh-push-to-registry-redhat-iotest
+  applications:
+    - rh-push-to-registry-redhat-iotest
   origin: dev-release-team-tenant
-  releaseStrategy: rh-push-to-registry-redhat-io-test-push-rs
+  pipelineRef:
+    params:
+      - name: url
+        value: https://github.com/redhat-appstudio/release-service-catalog.git
+      - name: revision
+        value: main
+      - name: pathInRepo
+        value: pipelines/rh-push-to-registry-redhat-io/rh-push-to-registry-redhat-io.yaml
+    resolver: "git"
+  policy: rh-push-to-registry-redhat-io-test-policy
+  serviceAccount: release-service-account
+  data:
+    pyxis:
+      server: stage
+      secret: pyxis
+    images:
+      addGitShaTag: true
+    mapping:
+      components:
```

Comment:
Awaiting reply from @happybhati on this one. The mapping in the ReleaseStrategy doesn't exist, so not sure how it worked or what the intended mapping is

Reply:
if it works as-is, then we are good.